README.rst

```diff
@@ -278,7 +278,7 @@ its status:
 
     $ reana-client workflow status
     NAME       RUN_NUMBER   CREATED               STATUS     PROGRESS   COMMAND
-    workflow   5            2018-07-12T06:33:20   finished   2/2        root -b -q '../code/fitdata.C(\"data.root\",\"plot.png\")'
+    workflow   1            2018-07-12T06:33:20   finished   2/2        root -b -q '../code/fitdata.C(\"data.root\",\"plot.png\")'
```

Comment:
This should be the output from QA? latest master looks a bit different, it is connected with https://github.com/reanahub/reana-workflow-engine-cwl/pull/49 discussion.

Reply:
Yes, it's the output of the moment the PR was done. The outputs will naturally change for v0.3.0.